rita_common/Cargo.toml

```diff
@@ -36,6 +36,7 @@ awc = {version = "3.0.0-beta.21", default-features = false, features=["openssl",
 actix-service = "2.0.2"
 web30 = "0.22"
 althea_types = { path = "../althea_types" }
+mac_address = "1.1.4"
```

Comment:
no need for an additional dependency https://serverfault.com/questions/357811/find-mac-addresses-in-proc-or-somewhere-despite-bonding-device

We can grab the mac address of the first interface by parsing proc. Easy to implement.

Reply:
Couldnt find proc/net/bonding, but found in /sys/class/net